entities/config.local.json

```diff
@@ -0,0 +1,28 @@
+{
+    "insights-dispatch": {
```

Comment:
do we want to have a more general pattern here for configs for modules?
maybe move everything down a level so that top level config can be reserved? thoughts?
```
{
  "insights-dispatch": {
    "module": ...,
    "config": {
      ..
    }
}